backend/src/services/combo-service.ts

```diff
@@ -142,6 +142,32 @@ class ComboService extends GeneralService<ICombo, IComboDto> {
       throw error;
     }
   }
+
+  async deleteItemsByServiceId(serviceId: string, t?: any): Promise<void> {
+    let allComboIds = await this.serviceComboModel.findAll({
+      where: {
+        service_id: serviceId
+      }
+    });
+    allComboIds = allComboIds.map((item: Model) => (item.getDataValue('combo_id')));
+    let counter = 0;
+    for (const comboId of allComboIds) {
+      await this.model.destroy({
+        where: {
+          id: comboId
+        },
+        transaction: t
+      });
+      counter++;
+    }
+    console.log(counter + 'combos has been deleted')
+    let leftComboIds = await this.serviceComboModel.findAll({
```

Comment:
It'd be good practice to use `const` whenever the variable doesn't change their value.

Reply:
agree, thanks for you reviewing